common/api/appui-react.api.md

```diff
@@ -4650,10 +4650,12 @@ export interface ToolbarPopupProps extends PopupPropsBase {
 
 // @beta
 export interface ToolbarProps extends CommonProps, NoChildrenProps {
+    enableOverflow?: boolean;
```

Comment:
Should we move `expandsTo` option to `enableOverflow` to make API non-ambiguous, since it doesn't make sense to provide `expandsTo` if `enableOverflow` is `false`?